### ISSUE TITLE
[FEATURE]Use all feature set to generate rule set

### ISSUE THREAD

**Issue by jaygrth** [scikit-multiflow/scikit-multiflow#214]
Hi,

I was wondering if it is possible to be able to include all features in a hoeffding tree classifier (and the variations) like it is possible to do so in scikit learn decision tree classifier. This would be valuable for creating a comprehensive rule set, even if a feature is not relevant to the accuracy of a classifier. 

Thanks

**Comment by jacobmontiel** [comment]
Hello @jaygrth 

This is not possible in our implementation as it is not part of the original algorithm. Could you provide more information about this option in scikit-learn?

**Comment by jaygrth** [comment]
Hi,

https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html

If you set max_features lower than the number of features, it forces the model to use all features.

**Comment by jacobmontiel** [comment]
From the documentation you shared:

> If `None`, then `max_features=n_features`

This means that all features are considered when searching for a split point. This is the same in the Hoeffding Tree classifier. By default, we consider all features when looking for a split. We do not support any other option. That being said, the use of a feature depends on its relevance to the given problem, even if all features are considered, it might be the case that some features are never used in the tree.

**Comment by jacobmontiel** [comment]
Closing with no action required.
